CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bugs fixed
 
+* [#429](https://github.com/clojure-emacs/clojure-mode/issues/429): Fix last occurrence of expression sometimes is not replaced when using `move-to-let`.
```

Comment:
This sentence sounds a bit strange to me. :-) Perhaps "Fix a bug causing the last..."?